modules/experimental-layers/src/bitmap-layer/bitmap-layer-fragment.js

```diff
@@ -0,0 +1,79 @@
+export default `
+#define SHADER_NAME bitmap-layer-fragment-shader
+
+#ifdef GL_ES
+precision highp float;
+#endif
+
+uniform sampler2D uBitmap0;
+uniform sampler2D uBitmap1;
+uniform sampler2D uBitmap2;
+uniform sampler2D uBitmap3;
+uniform sampler2D uBitmap4;
+uniform sampler2D uBitmap5;
+uniform sampler2D uBitmap6;
+uniform sampler2D uBitmap7;
+uniform sampler2D uBitmap8;
+uniform sampler2D uBitmap9;
+uniform sampler2D uBitmap10;
+
+varying vec2 vTexCoord;
+varying float vBitmapIndex;
+varying vec4 vPickingColor;
+varying vec4 vInstanceColor;
+
+uniform float desaturate;
+uniform vec4 transparentColor;
+uniform vec3 tintColor;
+uniform float opacity;
+
+// apply desaturation
```

Comment:
Follow-up PR: We should put the desaturation/tint in a separate little shader module.